ical: add page (#5849)
tldr-pages/tldr: pages/common/ical.md

# ical

> > Program to display english Calendar on your terminal

Reviewer: ```suggestion
> Program to display english Calendar on your terminal.
```
According to <https://manned.org/ical>, `ical` converts the given Gregorian date to a Hirji calendar.

Reviewer 2: >and vice-versa

> 

Reviewer: ```suggestion
More information: <https://manned.org/ical>.

```
We try to put links to man pages or home pages into every page. If you find any official link for `ical` feel free to replace my suggested link.

Reviewer 2: @marchersimon you frequently by mistake omit the `>` at the beginning ;)

Reviewer: Oh, thanks for reminding me

> - Display current month's calendar

Reviewer: ```suggestion
- Display the current month's calendar:
```

`ical`

> - Display the calendar of a a particular year

Reviewer: ```suggestion
- Display the calendar of a specific year:
```

`ical {{year eg: 1999}}`

> - Display the calendar for a particular  month of particular yar

Reviewer: ```suggestion
- Display the calendar for a specific month of a specific year:
```

# ical

> > Program to display english Calendar on your terminal.

Reviewer: ```suggestion
> A Hirji/Islamic calendar and converter for the terminal.
```

> More information: <https://manned.org/ical>.

Reviewer 2: ```suggestion
> More information: <https://manned.org/ical>.
```

- Display the current month's calendar:

`ical`

> - Display the calendar of a specific year:

Reviewer: ```suggestion
- Convert a Gregorian date to a Hijri date:
```

> `ical {{year eg: 1999}}`

Reviewer: ```suggestion
`ical --gregorian {{yyyymmdd}}`
```

> - Display the calendar for a specific month of a specific year:

Reviewer: ```suggestion
- Convert a Hirji date to a Gregorian date:
```

> `ical {{month, eg: 10}} {{year, eg: 1999}}`

Reviewer: ```suggestion
`ical --hijri {{yyyymmdd}}`
```
